ai_bridge/models/ai_bridge.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2025 Dixmit
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import json
+
+from odoo import api, fields, models
+from odoo.tools.safe_eval import safe_eval
+
+
+class AiBridge(models.Model):
+
+    _name = "ai.bridge"
+    _description = "Ai Bridge Configuration"
+    _order = "sequence, id"
+
+    sequence = fields.Integer(
+        default=10,
+    )
+    name = fields.Char(required=True)
+    active = fields.Boolean(default=True)
+    description = fields.Text()
+    model_id = fields.Many2one(
+        "ir.model",
+        string="Model",
+        required=True,
+        ondelete="cascade",
+        help="The model to which this bridge is associated.",
+    )
+    model = fields.Char(related="model_id.model", string="Model Name")
+    domain = fields.Char(
+        string="Filter", compute="_compute_domain", readonly=False, store=True
+    )
+    execution_ids = fields.One2many("ai.bridge.execution", "ai_bridge_id")
+    execution_count = fields.Integer(
+        compute="_compute_execution_count",
+    )
+    url = fields.Char(
+        string="URL",
+        help="The URL of the external AI system to which this bridge connects.",
+    )
+    auth_type = fields.Selection(
+        selection=[
+            ("none", "None"),
+            ("basic", "Basic Authentication"),
+            ("token", "Token Authentication"),
+        ],
+        default="none",
+        string="Authentication Type",
+        help="The type of authentication used to connect to the external AI system.",
+    )
+    group_ids = fields.Many2many(
+        "res.groups",
+        help="User groups allowed to use this AI bridge.",
+    )
+    field_ids = fields.Many2many(
+        "ir.model.fields",
+        help="Fields to include in the AI bridge.",
+        compute="_compute_field_ids",
+        store=True,
+        readonly=False,
+    )
+    auth_username = fields.Char(groups="base.group_system")
+    auth_password = fields.Char(groups="base.group_system")
+    auth_token = fields.Char(groups="base.group_system")
+    sample_payload = fields.Text(
+        help="Sample payload to be sent to the AI system. "
+        "This is used for testing and debugging purposes.",
+        compute="_compute_sample_payload",
+    )
+
+    @api.depends("model_id")
+    def _compute_domain(self):
+        for record in self:
+            record.domain = "[]"
+
+    @api.depends("model_id")
+    def _compute_field_ids(self):
+        for record in self:
+            record.field_ids = False
+
+    @api.depends("field_ids", "model_id")
```

Comment:
Do you think it's necessary to reset the sample value every time a field is specified? You might need to add an extra field, and that would eliminate the configuration already set up. I would only delete it if the model changes. What do you think?

Reply:
this payload is a sample that will be used for testing. I don't see the problem.

Also, this paayload will never be used to connect. It is useful only from visibility perspective.